ush/ufsda/misc_utils.py

```diff
@@ -80,9 +81,25 @@ def create_batch_job(job_config, working_dir, executable, yaml_path):
 cd {working_dir}
 """
         f.write(commands)
-        if scheduler[job_config['machine']] == 'slurm':
-            f.write(f"srun -n $SLURM_NTASKS {executable} {yaml_path}\n")
+        if single_exec:
+            # standalone jedi application
+            if scheduler[job_config['machine']] == 'slurm':
+                f.write(f"srun -n $SLURM_NTASKS {executable} {yaml_path}\n")
+        else:
+            # TODO (Guillaume): Hard coded for soca in a few places, change that
```

Comment:
Just making a note to resolve this sooner rather than later.

Reply:
@CoryMartin-NOAA  On my todo list as soon as I can cycle mom6-cice6 in the global-workflow.

Reply:
see issue #100